README.md

```diff
@@ -58,152 +104,394 @@ Options
    --no-check-duplicates       do not check messages for duplicates
 ```
 
-## Environment variables
+#### `--version` (alias: `-v`)
```

Comment:
How this part is better then `—help`? Maybe let’s keep `—-help`, so there was one point of truth?